apps/build-onchain-apps/src/components/disclaimer/Disclaimer.tsx

```diff
@@ -0,0 +1,32 @@
+import Image from 'next/image';
+
+export default function Disclaimer() {
+  return (
+    <div className="mt-28 flex w-full flex-col items-center justify-between rounded-xl bg-[#141519] p-4 text-base text-white md:mt-12 md:flex-row lg:mt-24">
```

Comment:
A couple notes:
1. add the bg color in the extra colors in the tailwind configuration https://github.com/coinbase/build-onchain-apps/blob/main/apps/build-onchain-apps/tailwind.config.ts#L32 as makes them easier to reuse and understand
2. you can split this long line with `clx` as you can see here https://github.com/coinbase/build-onchain-apps/blob/main/apps/build-onchain-apps/src/components/header/AccountDropdown.tsx#L29 as that will make the css more readiable.

Reply:
Makes sense! Updated this